### DISCUSSION TITLE
Embedding in GTK Apps

### DISCUSSION THREAD

**Discussion by emmyoh** [servo/servo#31498] [General]
Hello,

Wanted to ask if embedding Servo into a GTK app (like WebKitGTK) is currently possible or planned? For what I'm thinking, support for implementing custom URI schemes (ie, adding protocols beyond HTTP) asynchronously (ie, fetching data incrementally without blocking the main thread) would be of particular interest.

Thanks!

**Comment by jdm** [comment]
I had a proof of concept in https://github.com/jdm/servo-embedding-example that should be updated.
